style.css

```diff
@@ -1,455 +1,1107 @@
+  width: 327px;
+  left: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-top: -20px;
 }
 
-.leo {
-  align-items: center;
-  justify-content: space-between;
+form {
   display: flex;
-  float: left;
-  margin-bottom: 1px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: -39px;
 }
 
-.leo .text {
-  margin-right: 200px;
+ul {
+  list-style-type: none;
 }
 
-.ingine {
+#name {
+  border-radius: 8px;
+  height: 40px;
+  width: 305px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
   align-items: center;
-  justify-content: space-between;
+  color: #172b4d;
+}
+
+#email {
+  height: 40px;
+  width: 305px;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
   display: flex;
-  float: left;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 18px;
 }
 
-.ingine .text {
-  margin-right: 150px;
-  justify-content: center;
+#msg {
+  margin-top: 18px;
+  width: 305px;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  height: 160px;
+  resize: none;
 }
 
-.see-project:hover {
-  background-color: #6070ff;
-  text-decoration: underline;
-  text-decoration-color: #396df2;
-  color: white;
+input {
+  background: #fff;
+  border: 1px solid #cfd8dc;
 }
 
-.getmy:hover {
-  color: white;
+#get-in-touch {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  margin-top: 12px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+#get-in-touch:hover {
   background-color: #6070ff;
-  text-decoration: underline;
-  text-decoration-color: #396df2;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  color: #fff;
+}
+
+#get-in-touch:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+@media screen and (min-width: 768px) {
+  html,
+  body {
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .sijijui img {
+    width: 8px;
+    height: 8px;
+  }
+
+  header {
+    display: flex;
+  }
+
+  .navbar .humburger {
+    display: none;
+  }
+
+  #nav-items {
+    display: flex;
+    gap: 15%;
+  }
+
+  .nav-item {
+    display: inline-block;
+    gap: 15%;
+  }
+
+  .nav-item a {
+    font-size: 1rem;
+    color: #000;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 1.2rem;
+  }
+
+  .header-info-secion {
+    background-image: url(img/Header\ bg.png);
+    background-size: cover;
+  }
+
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you implemented a few changes as per the previous code reviewer's request 👍🏻 but not complete your design has too much gap or spacing at the right-hand-side at `=768px` screen size, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![template_1_hero-section](https://github.com/Edgahkipkemoi/mobile/assets/66476329/debf459f-2a30-4ce2-92ae-a0e012d6e707)

</td>
<td>

![Screenshot from 2023-06-17 17-28-10](https://github.com/Edgahkipkemoi/mobile/assets/66476329/3ba3ca70-c905-43d4-b12c-233d0a445205)

</td>
</tr>
</table>